lib/decidim/question_captcha/engine.rb

```diff
@@ -32,6 +32,12 @@ class Engine < ::Rails::Engine
           Decidim::RegistrationForm.class_eval do
             extend(ActsAsTextcaptcha::Textcaptcha)
             include(Decidim::QuestionCaptcha::HasCaptcha)
+
+            validate :validate_textcaptcha, if: :perform_textcaptcha?, on: :validate_captcha
```

Comment:
Could you move this line to `Decidim::QuestionCaptcha::HasCaptcha`?

Reply:
It should be ok now @armandfardeau